pulsar-broker/src/test/java/com/yahoo/pulsar/common/naming/ServiceConfigurationLoaderTest.java

```diff
@@ -26,17 +26,17 @@
 
 import org.testng.annotations.Test;
 
-import com.yahoo.pulsar.broker.FieldContext;
 import com.yahoo.pulsar.broker.ServiceConfiguration;
-import com.yahoo.pulsar.broker.ServiceConfigurationLoader;
+import com.yahoo.pulsar.common.configuration.FieldContext;
+import com.yahoo.pulsar.common.configuration.PulsarConfigurationLoader;
 
 public class ServiceConfigurationLoaderTest {
```

Comment:
I think this class should be renamed to `PulsarConfigurationLoaderTest`, because `ServiceConfigurationLoader` is renamed to `PulsarConfigurationLoader`.

Also, this class was a test class for `ServiceConfigurationLoader` but not for `ServiceConfiguration`. I guess this naming mismatch causes a misread, which is the reason why some of test methods are (incorrectly) simply creating a `ServiceConfiguration` instance without PulsarConfigurationLoader.